repos.go

```diff
@@ -282,6 +282,59 @@ func recentReleases(count int) []Repo {
 	return repos
 }
 
+/*
+	{
+		repositoryOwner(login: "charmbracelet") {
+		id
+		login
+		repositories(
+			first: 5
+			privacy: PUBLIC
+			orderBy: {field: PUSHED_AT, direction: DESC}
+		) {
+			edges {
+			node {
+				name
+				description
+				url
+			}
+			}
+		}
+		}
+	}
+*/
+func recentPushes(owner string, count int) []Repo {
+	var query struct {
+		Owner struct {
+			Repositories struct {
+				Edges []struct {
+					Node qlRepository
+				}
+			} `graphql:"repositories(first: $count, privacy: PUBLIC, orderBy: {field: PUSHED_AT, direction: DESC})"`
+		} `graphql:"repositoryOwner(login: $owner)"`
+	}
+	fmt.Printf("Finding repos with recent pushes in the %s org\n", owner)
+	var repos []Repo
+	variables := map[string]interface{}{
+		"count": githubv4.Int(count),
+		"owner": githubv4.String(owner),
+	}
+	err := gitHubClient.Query(context.Background(), &query, variables)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, v := range query.Owner.Repositories.Edges {
+		// ignore meta-repo
+		repos = append(repos, repoFromQL(v.Node))
```

Comment:
You're _not_ ignoring anything here 😄 

```suggestion
		if string(v.Node.NameWithOwner) == fmt.Sprintf("%s/%s", username, username) {
			continue
		}
		
		repos = append(repos, repoFromQL(v.Node))
```

Reply:
Oh lol! I forgot to delete the comment. I think it's okay if we include the meta repo in our recent activity 🤔 What do you think?